CHANGELOG.md

```diff
@@ -9,6 +9,13 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 [Unreleased][Unreleased_log]
 --------------
+### Added
+- In Linux, 0-base enclave creation is enabled.
+  - Developers can create an enclave with base address 0x0 by either defining SGX_ENCLAVE_ZERO_BASE (before including headers) or
+    using the oesign tool config file option 'CreateZeoBasedEnclave'.
+  - Optionally, the enclave image start address can be provided by assigning the value to SGX_ENCLAVE_START_ADDRESS or via
+  config file option 'StartAddress'.
+  - tests/SampleAppZeroBase provides an example of 0-base enclave creation.
```

Comment:
Tests are not published, and the user will not have access to the tests upon installing OE SDK.
It would be better to create a sample for zero-base enclaves.